easybuild/easyconfigs/c/CHASE/CHASE-v1.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'CHASE'
+version = 'v1'
```

Comment:
@rtripath89 wouldn't it make more sense to use a datestamp as a version, i.e. the data of the last update?